pom.xml

```diff
@@ -1339,5 +1339,26 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>disableWizard</id>
+      <activation>
+        <property>
+          <name>!enableWizard</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jenkins-ci.tools</groupId>
+            <artifactId>maven-hpi-plugin</artifactId>
+            <configuration>
+              <systemProperties>
+                <jenkins.install.state>TEST</jenkins.install.state>
```

Comment:
This differs from [`hudson.Main.development` used by core `hudson-dev:run`](https://github.com/jenkinsci/jenkins/blob/df21a13cd6d86c01381bd79f31894e98a79f3e90/war/pom.xml#L530). We should use the same flag, or set it to `DEVELOPMENT` which comes to the same thing. (I do not know what the behavioral difference would be, if any.)